server/build.gradle

```diff
@@ -29,6 +28,27 @@ micronaut {
     }
 }
 
+graalvmNative {
+    toolchainDetection = false
+    binaries {
+        all {
+            imageName.set('check-ins')
+
+            // See https://github.com/micronaut-projects/micronaut-core/issues/10771
+            buildArgs.add("--initialize-at-build-time=kotlin.coroutines.intrinsics.CoroutineSingletons")
+
+            // When we are building on CI, we do not want to include the dev migrations.
+            def isCI = System.getenv("CI") != null
+            def migrationLocations = isCI ? "db/common" : "db/common,db/dev"
```

Comment:
What if we are building for the dev environment? We run the dev migrations there.

Reply:
Good call...

I'm wondering whether we should pivot to simply use `dockerBuildNative` on CI to generate the docker image automatically for us...

I've pushed a commit which does this (and tests the image building works on CI), and also switched to check the `SERVICE_NAME` and var (and disable the dev db load when it exists and equals `checkins-master`)